addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -7843,6 +7843,44 @@ QUnit.module("Views", (hooks) => {
         );
     });
 
+    QUnit.test("can save without any dirty translatable fields", async function (assert) {
+        serverData.models.partner.fields.foo.translate = true;
+        serverData.models.partner.fields.display_name.translate = true;
+
+        patchWithCleanup(localization, {
+            multiLang: true,
+        });
+
+        await makeView({
+            type: "form",
+            resModel: "partner",
+            serverData,
+            arch: `
+                <form>
+                    <sheet>
+                        <group>
+                            <field name="foo"/>
+                            <field name="display_name"/>
+                        </group>
+                    </sheet>
+                </form>`,
+            resId: 1,
+            mockRPC(route, args) {
+                assert.step(`${args.method}`);
+            },
+        });
+
+        assert.verifySteps(["get_views", "read"]);
```

Comment:
assert that the fields have the translatable feature (I guess there's a classname or something)